domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/event/AbstractShowConditionValidator.java

```diff
@@ -49,7 +63,8 @@ protected void validateShowConditionFields(EventEntity eventEntity,
                 if (!forShowConditionFieldExistsAtLeastOneCaseFieldEntity(
                     showConditionField,
                     eventEntity.getCaseType().getCaseFields())
-                    && !MetadataField.isMetadataField(showConditionField)) {
+                    && !(MetadataField.isMetadataField(showConditionField)
+                            || isInjectedField(showConditionField, isSupportingInjectedData))) {
```

Comment:
```suggestion
                    && !MetadataField.isMetadataField(showConditionField) 
                    && !isInjectedField(showConditionField, isSupportingInjectedData)) {
```
`!(A || B)` = `!A && !B`, I think the latter is a bit more readable!

Reply:
Sure - makes sense - I had copied the approach from elsewhere

Will fix

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/1203/commits/1a9c18091b83dd43eaf785f1bb6543fe7b24d22c